hopp/offshore/fixed_platform.py

```diff
@@ -30,8 +36,9 @@
 """
 ''' 
 Notes:
-    Thank you Jake Nunemaker's oswh2 repository!!!
-    pile_cost=0 $US/tonne for monopile construction. Not a bug, this # is consistent with the rest of ORBIT
+    - Thank you Jake Nunemaker's oswh2 repository!!!
```

Comment:
Can you point me to this repository? It sounds helpful.

Reply:
It was just a private repository that Jake shared with me from a previous offshore-h2 analysis. We'd have to reach out to him for access.